libraries/AP_OSD/AP_OSD.h

```diff
@@ -123,6 +127,16 @@ class AP_OSD_Screen
     AP_OSD_Setting bat2_vlt{false, 0, 0};
     AP_OSD_Setting bat2used{false, 0, 0};
     AP_OSD_Setting clk{false, 0, 0};
+    
+    // MSP OSD only
```

Comment:
Should these be gated behind MSP being enabled?

Reply:
perhaps but how?